BattleTank/PowerUp.cs

```diff
@@ -76,8 +77,15 @@ public virtual void Update(GameTime gameTime)
                             game.tank1.mines++;
                             break;
                         case MATRIX:
-                            game.frozenPlayer = 1;
-                            game.timer3control = 1;
+                            if (game.enemyTanks.Count != 0)
+                            {
```

Comment:
Foreach nie potrzebuje sprawdzenia, czy co kolwiek ma, to jest jego zaleta. Jak lista jest pusta, to nic nie zrobi :).

Reply:
Super! :D